task-definition.plugins-async.json

```diff
@@ -0,0 +1,193 @@
+                },
+                {
+                    "name": "KAFKA_CLIENT_CERT_B64",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:KAFKA_CLIENT_CERT_B64::"
+                },
+                {
+                    "name": "KAFKA_CLIENT_CERT_KEY_B64",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:KAFKA_CLIENT_CERT_KEY_B64::"
+                },
+                {
+                    "name": "KAFKA_TRUSTED_CERT_B64",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:KAFKA_TRUSTED_CERT_B64::"
+                },
+                {
+                    "name": "KAFKA_HOSTS",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:KAFKA_HOSTS::"
+                },
+                {
+                    "name": "POSTHOG_PERSONAL_API_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:POSTHOG_PERSONAL_API_KEY::"
+                },
+                {
+                    "name": "REDIS_URL",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:REDIS_URL::"
+                },
+                {
+                    "name": "SENTRY_DSN",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SENTRY_DSN_PLUGIN_SERVER::"
+                },
+                {
+                    "name": "JOB_QUEUE_GRAPHILE_URL",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:JOB_QUEUE_GRAPHILE_URL::"
+                },
+                {
+                    "name": "STATSD_HOST",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STATSD_HOST::"
+                }
+            ],
+            "entryPoint": ["./bin/plugin-server", "--no-restart-loop"],
+            "ulimits": [
+                {
+                    "name": "nofile",
+                    "softLimit": 1024000,
+                    "hardLimit": 1024000
+                }
+            ],
+            "linuxParameters": {
+                "initProcessEnabled": true
+            },
+            "healthCheck": {
+                "retries": 10,
+                "command": ["CMD-SHELL", "curl -f http://localhost:6738/_health || exit 1"],
+                "timeout": 30,
+                "interval": 20,
+                "startPeriod": 60
+            }
+        }
+    ],
+    "requiresCompatibilities": ["FARGATE"],
+    "cpu": "4096",
+    "memory": "16384"
```

Comment:
happy to leave for now but how about we lower these for both plugins and plugins-async

Reply:
I love this idea. Just keep in mind that we don't have a great distribution of work over the plugin server tasks so if you scale down vertically our max throughput will also decrease:
![image](https://user-images.githubusercontent.com/391319/170387896-d64b4f64-b949-42e3-88bd-554b3eb63929.png)

Reply:
Let's do this separately unless we know what these numbers should be.